README.md

```diff
@@ -5,3 +5,61 @@
+The extra cost of staff (2 developers for 1) is very well compensated by the increase in development quality which allows to detect errors earlier. Indeed, the later a bug is detected, the more expensive it is to correct it.
+
+**Communication and Team spirit**
+
+Pair programming is a social activity that requires the ability to work in a team. It allows to develop the communication skills of the different collaborators to form a strong team spirit.
+This team spirit significantly improves the well-being of the employees in their work and therefore the productivity and quality of the work provided. Indeed, many surveys have shown that more than 90% of developers prefer to program in pairs rather than alone.
+
+
+**Indicators of non-performance**
+
+Beware of non-performance indicators that can be a brake on au pair programming. When training pairs, it is important to choose the employees who will work together according to the objectives of the activity.
+
+* Silence: Pair programming requires you to program aloud and share your point of view with your partner. Persistent silence indicates a lack of collaboration.
+* Disengagement: One member loses interest in the project and goes about his or her business.
+* Deletion: When one member is more experienced than another, the novice is content to watch the expert perform the majority of the development tasks.
+* Relationship problems: the two members of the pair do not get along and do not wish to work together.
+
+
+Pair programming is therefore a good alternative to development alone. It allows the development of communication skills that are essential for teamwork. In companies, it is also used to quickly transfer skills.
+
+[Read More](https://en.wikipedia.org/wiki/Pair_programming)
+
+---
+
+###Checklist
+
+**The importance of checklists**
+
+Checklists, in support of development and organization tools, are an essential tool for project monitoring and task completion. They allow you to note all the important points and discuss / carry out on the project during one day for example.
+
+**Checklists for code reviews**
+
+It is important to address this point in the coding review section because this tool can also be used for code reviews.
+You can write down all the points to be discussed, the important things to check. Once a task is done, just check the corresponding box. It's also a tool that will free space in its memory and allow us to concentrate on the essential tasks in the work. In fact, instead of keeping in the corner of our head all the tasks we have to do, we write them down as we think about them and thus free up space in our brain to concentrate at best.
+
+**Checklists Tools**
+
+There are a large number of tools for making checklists called ToDoApps. On a smartphone or computer, it is up to you to choose the tool that suits you best.
+
+Here is a non-exhaustive list of several applications:
+
+* Any.do
+* Todoist
+* Microsoft To-do
+* Google Task
+
```

Comment:
Good examples ! Maybe it would be interesting to evoke the automation feature like in Github Projects or Trello which are integrated with PR and issues i.e automatic card creation